### PR TITLE
[ZEPL-2187] Guide user use '=' in conf interpreter

### PR DESCRIPTION
**Pull request by Leemoonsoo**
This PR is followup documentation of https://github.com/ZEPL/zeppelinhub/pull/6202.

 - Guide user uses '=' 
 - Available configurations.

### CODE REVIEW

File: docs/guide/interpreter/config.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Note that inline configuration must run before any other paragraph. So it will u
 
 Key can not include empty space. `[A-Z_0-9]+` is treated as a environment variable, otherwise it's considered an interpreter property. e.g.:
 
- - `spark.app.name`, `python.python` - Interpreter property
+ - `spark.app.name`, `zeppelin.python` - Interpreter property
```
Comment (1ambda) [inline docs/guide/interpreter/config.md:24]: Would `zepl.python` be better for advertisement? if the value doesn't have effect on the runtime.

Comment (1ambda) [reply docs/guide/interpreter/config.md:24]: oh. seems it's being used. :(

File: docs/guide/interpreter/config.md [hunk 2]
```diff
@@ -31,15 +31,16 @@ Value is separated by spaces or tabs from Key. Value can include any character e
  - `100`, `true`, `/dir/path`, `spark app name`
 
 
+
 ### Example
```
Comment (1ambda) [inline docs/guide/interpreter/config.md:36]: I think we can add some description that If you put some value doesnt include `.` (dot), they can be used as env variable in the paragraph.

**Review by 1ambda — APPROVED** [review]
left few suggestions.

File: docs/guide/interpreter/config.md (hunk 1 above)
Comment (Leemoonsoo) [reply docs/guide/interpreter/config.md:24]: I would, but the name of properties are depends on implementation of each interpreter, which property they reads.

ConfInterpreter can try to rewrite property name, like
`zepl.xxxx.yyyy` --> `zeppelin.xxxx.yyyy`

But that will be separate work.

File: docs/guide/interpreter/config.md (hunk 2 above)
Comment (Leemoonsoo) [reply docs/guide/interpreter/config.md:36]: [Previous section](https://github.com/ZEPL/zepl-documentation/pull/52/files#diff-f2c3433572feaba6dbdf97abb0479263L22) explains it. Do you think it's unclear?

**Pull request merged**